src/libraries/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/ICertificatePal.cs

```diff
@@ -27,5 +27,7 @@ internal interface ICertificatePal : ICertificatePalCore
         ICertificatePal CopyWithPrivateKey(DSA privateKey);
         ICertificatePal CopyWithPrivateKey(ECDsa privateKey);
         ICertificatePal CopyWithPrivateKey(RSA privateKey);
+        bool GetPolicyData(out byte[] ApplicationCertPoliciesData, out byte[] CertPoliciesData, out byte[] CertPolicyMappingsData,
+                           out byte[] CertPolicyConstraintsData, out byte[] EnhancedKeyUsageData, out byte[] InhibitAnyPolicyExtensionData);
```

Comment:
Parameter names should be camelCased, and please chop to one per line.

```C#
bool GetPolicyData(
    out byte[] applicationPolicies,
    out byte[] certPolicies,
    ...
    out byte[] inhibitAnyPolicy);
```

Reply:
Alternatively, consider defining a PolicyData struct and making the interface method be `PolicyData GetPolicyData()`.  It's internal, so just internal fields, no properties needed.

Reply:
would that be preferable? I generally preserved as much as I could but I think structure instead of large number of parameters may be more manageable.

Reply:
Once it's being added as an interface method I think the structure makes more sense than a lot of outs.

Reply:
updated. I do like the struct better and it will be easier to update if we need to.